src/PatientChecking/PatientChecking/Services/PatientService.cs

```diff
@@ -8,5 +10,25 @@ namespace PatientChecking.Services
 {
     public class PatientService : IPatientService
     {
+        private readonly PatientCheckInContext patientCheckInContext;
```

Comment:
coding convention: always use _  prefix for private fields

Reply:
"coding convention: always use _ prefix for private fields" has been applied

Reply:
I don't see the fix being applied ?